test/distributed/test_ucc.py

```diff
@@ -0,0 +1,103 @@
+# Owner(s): ["oncall: distributed"]
```

Comment:
Does this file need to be exposed to OSS? Or is the intention indeed trigger OSS c10d tests using UCC backend?